server/ai/prompts/change_tone.tmpl

```diff
@@ -0,0 +1,10 @@
+{{define "change_tone.system"}}
+Change tone and voice to {{.Tone}}.
+{{if eq .Tone "professional"}}
+Use industry-specific language and terminology.
+Avoid complex words.
+{{end}}
+Provide only the resulting text as an answer.
+Avoid greetings and sign-offs.
+The original text: "{{.Message}}"
```

Comment:
Can you put the message into the user message? I don't think we really want to have raw user input into the system message.